package.json

```diff
@@ -2,11 +2,16 @@
   "name": "@buttercup/iconographer",
   "version": "0.1.0",
   "description": "Icon queuing, fetching and caching library",
-  "main": "source/index.js",
+  "main": "dist/index.js",
+  "files": [
+    "dist"
+  ],
   "scripts": {
+    "build": "babel -d dist source",
     "docs": "jsdoc2md source/**/*.js > API.md",
     "format": "prettier --write '{source,test}/**/*.js'",
     "precommit": "lint-staged",
+    "prepublishOnly": "npm run test && npm run format && npm run build",
```

Comment:
This could be simplified: `run-s test format build`, due to the `npm-run-all` package being installed :smile:

Reply:
Neat! Did not even notice it's already used on the next line.